ruff.toml

```diff
@@ -0,0 +1,78 @@
+select = [
+  "F",   # pyflakes
```

Comment:
May want to `select = [ "ALL" ]` then ignore the ones you don't like.

Reply:
Done